README.md

```diff
@@ -1,180 +1,143 @@
-$ napi init
-$ napi ui
-```
+## Documentation
 
-## Examples
+Comprehensive documentation is available on our [documentation website](https://nanoapi.io/docs/nanoapi).
 
-We are building a collection of example API repositories in each language so you can explore the project. This list will grow over time.
-
-[examples](/examples/README.md)
+## Quick Start
 
-## Usage
+Ensure you have Node.js (>=18) and npm installed.
 
 ### Installation
 
-We are working on a binary release, but for now, you can install it via this github repository:
-
 ```bash
-$ git clone https://github.com/nanoapi-io/napi.git
-$ cd napi
-$ npm install
+npm install -g @nanoapi.io/napi
 ```
 
-### Usage
-
-To open the UI and inspect your codebase, run the following commands:
+### Getting Started
 
 ```bash
-$ napi init
-$ napi ui
+napi init
+napi ui
 ```
 
-### Commands:
+This will initialize a new NanoAPI project and open the UI for inspecting and refactoring your codebase.
 
-```
-  init                  Initialize the NanoAPI project
-  ui [entrypoint]       Inspect the codebase and understand the API endpoints, middleware, and other API-specific code
-  split <entrypoint>    Transform the codebase into smaller, more manageable pieces at build time
-```
+## CLI Usage
 
-## Using the UI
+`napi` provides the following commands:
 
-The easiest way to refactor your API endpoints is to do it through our UI. You can group endpoints and create builds from here.
+```bash
+init                  Initialize the NanoAPI project
+ui [entrypoint]       Open the UI to inspect and refactor the codebase
+split <entrypoint>    Transform the codebase into microservices
+napi <platform> annotate <entrypoint> [--token=<key>] Automatically generate annotations for the codebase via OpenAI
+```
+
+For more detailed information about the CLI and what each command does, refer to our [CLI guide](https://nanoapi.io/docs/cli/).
 
-## Building with Annotations
+## Refactoring with Annotations
 
-You can also refactor your codebase by adding annotations to your code. Here is an example of how you can do it:
+You can use annotations to specify how to split your code. Here’s an example:
 
 ```typescript
 // src/api.js
 
-import * as express from "express";
-
-import * as service from "./service.js";
-
-const app = express();
-const port = 3000;
-
-app.use(express.json());
-
-// @nanoapi endpoint:/api/v2/maths/time method:GET group:Time
-app.get("/api/v2/maths/time", (req, res) => {
-  const result = service.time();
-  return res.json({
-    result,
-  });
-});
-
-// @nanoapi endpoint:/api/v2/maths/addition method:POST group:Maths
-app.post("/api/v2/maths/addition", (req, res) => {
-  const { body } = req;
-  const result = service.addition(body.a, body.b);
-  return res.json({
-    result,
-  });
+// @nanoapi endpoint:/api/v1/users method:GET group:Users
+app.get("/api/v1/users", (req, res) => {
+  res.send("User data");
 });
 
-// @nanoapi endpoint:/api/v2/maths/subtraction method:POST group:Maths
```

Comment:
I simplified the code here since it was mostly complicating things at the first glance at the README. We can try more engaging methods like a short  2 minute video to showcase the input/output flow and how the resulting microservices look like.

Reply:
This can of course be done in a separate PR since a video or a series of screenshots to demo the functionality need their own planning.